CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.9.0
+
+- Update grafana-aws-sdk version
+- Security: Upgrade Go in build process to 1.20.4
+- Revert grafana-plugin-sdk-go version to 0.149.0 to fix connection issues. Should be same behavior as behavior with no known issues.
```

Comment:
Oops I probably left that typo in my original changelog edit -- but if we upgrade, we won't need that part!

Reply:
removed!